ViAn/Project/Test/projecttestsuite.cpp

```diff
@@ -41,10 +43,14 @@ void ProjectTestsuite::add_remove_report_test(){
     proj->remove_report(idx3);
 
     QCOMPARE(proj->m_reports.size(), unsigned(0));
+
+    delete proj;
 }
 
 
 void ProjectTestsuite::save_load_test(){
+    // TODO Fix
```

Comment:
I looked a little at theses test cases and they all created a temporary dir but never removed it which eventually gave me error because i reach the max limit. The added deletes removed the temp folder except in this one. I know what the problem is and the solution lies in the tag-drawing branch. I'll fix it after these all are merged and just added a reminder.